pkg/pb/gracefulexit.proto

```diff
@@ -10,8 +10,41 @@ import "orders.proto";
 
 package gracefulexit;
 
+service NodeGracefulExit {
```

Comment:
Let's comment above this to clarify that this is for the storagenode private rpc service, since we have both satellite and storagenode protobufs in this file.